apps/fm/js/fm.js

```diff
@@ -918,11 +918,18 @@ function init() {
   );
 }
 
-window.addEventListener('load', function(e) {
-  AirplaneModeHelper.ready(function() {
-    airplaneModeEnabled = AirplaneModeHelper.getStatus() == 'enabled';
-    init();
-
+navigator.mozL10n.once(function(e) {
```

Comment:
are you saying that we should ask people to check readyState and set eventlistener, that then they should remove instead of using mozL10n.once which does exactly that? I don't understand...